backend/de.metas.fresh/de.metas.fresh.base/src/main/sql/postgresql/system/70-de.metas.fresh/5746180_sys_gh20034_Update_M_Warehouse_Org.sql

```diff
@@ -0,0 +1,18 @@
+-- Run mode: SWING_CLIENT
+
+-- Table: M_Warehouse
+-- 2025-02-10T08:44:20.589Z
+
+
+ALTER TABLE M_Warehouse
+    ADD CONSTRAINT M_Warehouse_AD_Org_Check
+        CHECK (AD_Org_ID > 0)
+;
+
+COMMENT ON CONSTRAINT M_Warehouse_AD_Org_Check ON M_Warehouse
+    IS 'Ensures that AD_Org_ID cannot be set to 0 to prevent MD_Stock_Update_From_M_HUs process to fail.'
+;
+
+UPDATE AD_Table SET TechnicalNote='M_Warehouse_AD_Org_Check constraint ensures that AD_Org_ID cannot be set to 0 to prevent MD_Stock_Update_From_M_HUs process to fail.',Updated=TO_TIMESTAMP('2025-02-10 08:44:20.430000','YYYY-MM-DD HH24:MI:SS.US')::timestamp without time zone AT TIME ZONE 'UTC',UpdatedBy=100 WHERE AD_Table_ID=190
```

Comment:
```suggestion
UPDATE AD_Table SET TechnicalNote='An additional DB-contraint named M_Warehouse_AD_Org_Check constraint ensures that AD_Org_ID cannot be set to 0 to prevent MD_Stock_Update_From_M_HUs process to fail.',Updated=TO_TIMESTAMP('2025-02-10 08:44:20.430000','YYYY-MM-DD HH24:MI:SS.US')::timestamp without time zone AT TIME ZONE 'UTC',UpdatedBy=100 WHERE AD_Table_ID=190
```